DOCSP-29503-cross-version-sync (#159)
mongodb/docs-cluster-to-cluster-sync: source/includes/fact-minimum-versions.rst

some older (pre-6.0) releases of MongoDB Server. This feature requires
additional preparation and configuration in pre-6.0 releases.

> Also, ``mongosync`` can perform a cross version migration

Reviewer: I think we don't need the change here. On the limitation page, we already have a paragraph talking about cross-version migration from `cross-version-sync.rst`

from a lower major version source cluster to a higher major version
destination cluster, up to two major versions ahead. For example,
migrate from 6.0 to 7.0. This also requires additional
preparation.

If you would like to migrate pre-6.0 versions of MongoDB Server using
``mongosync``, or perform a cross version migration, please `contact
<https://mongodb.com/contact>`__ your account team to inquire about

> Professional Services.

Reviewer: I think we can keep the original paragraph, since `If you would like to migrate pre-6.0 versions of MongoDB Server` also includes cross version migrate a pre-6.0 version.

some older (pre-6.0) releases of MongoDB Server. This feature requires
additional preparation and configuration in pre-6.0 releases.

> If you would like to migrate pre-6.0 versions of MongoDB Server using
> ``mongosync``, or perform a cross version migration, please `contact

Reviewer: ```suggestion
If you would like to migrate from a pre-6.0 versions of MongoDB Server using
``mongosync`` please `contact
```

I think we can remove the `perform a cross version migration` since we just want users to contact professional service when the source is < 6.0 in the cross version case not all the cases. `migrate from a pre-6.0 versions of MongoDB Server` should cover all the cases including cross-version.

Author: Will do.
